library/src/commonMain/kotlin/me/saket/swipe/SwipeableActionsBox.kt

```diff
@@ -125,11 +125,11 @@ private fun ActionIconBox(
         layout(width = placeable.width, height = placeable.height) {
           // Align icon with the left/right edge of the content being swiped.
           val iconOffset = if (action.isOnRightSide) constraints.maxWidth + offset else offset - placeable.width
-          placeable.placeRelative(x = iconOffset.roundToInt(), y = 0)
+          placeable.place(x = iconOffset.roundToInt(), y = 0)
         }
       }
       .background(color = backgroundColor),
-    horizontalArrangement = if (action.isOnRightSide) Arrangement.Start else Arrangement.End,
+    horizontalArrangement = if (action.isOnRightSide) Arrangement.Absolute.Left else Arrangement.Absolute.Right,
```

Comment:
Let's add a screenshot test to verify this? Here are some examples: https://github.com/saket/swipe/blob/trunk/library/src/test/kotlin/me/saket/swipe/SwipeableActionsBoxTest.kt#L43

Reply:
Not sure if I did this correctly. Please take another look!

Reply:
Yep this looks good, thank you!